toqito/matrix_ops/perturb_vectors.py

```diff
@@ -0,0 +1,34 @@
+"""Perturb vectors is used to add a small random number to each element of a vector.
+
+A random value is added sampled from a normal distribution scaled by `eps`.
+"""
+
+import numpy as np
+
+
+def perturb_vectors(vectors: list[np.ndarray], eps: float = 0.1) -> list[np.ndarray]:
+    """Perturb the vectors by adding a small random number to each element.
+
+    :param vectors: List of vectors to perturb.
+    :param eps: Amount by which to perturb vectors.
+    :return: Resulting list of perturbed vectors by a factor of epsilon.
+
+    Example:
+    ==========
+
+        >>> from toqito.matrix_ops import perturb_vectors
+        >>> import numpy as np
+        >>> vectors = [np.array([1.0, 2.0]), np.array([3.0, 4.0])]
+        >>> perturb_vectors(vectors, eps=0.1) # doctest: +SKIP
+
```

Comment:
@Shivansh20128 You have to add the output manually here. Right now, the RTD output is blank for the example.

![image](https://github.com/user-attachments/assets/45153088-6d96-45f8-9d46-2216bce6cc82)



See below for example:

https://github.com/vprusso/toqito/blob/9b355a57c98c83e4d6342f28d1b396b27bcf8abb/toqito/rand/random_povm.py#L21-L34

![image](https://github.com/user-attachments/assets/0a731784-bce0-40b3-9063-376ddf333923)

Reply:
Added! I hope its fine now.